site/src/content/docs/docs/integrations/api/graphql.mdx

```diff
@@ -0,0 +1,216 @@
+---
+title: "GraphQL"
+sidebar:
+  order: 1
+---
+Unionlabs provides a hosted indexing service for all connected chains, served at [graphql.union.build](https://graphql.union.build/v1/graphql). 
+A [UI](https://cloud.hasura.io/public/graphiql?endpoint=https://graphql.union.build/v1/graphql) is also provided, which can be used 
+to explore all available data. This guide goes through the most useful queries and types.
+
+Although most HTTP clients can be used to query our API, we recommend [URQL](https://commerce.nearform.com/open-source/urql/). Our GraphQL schema can
+be downloaded using [`graphqurl`](https://github.com/hasura/graphqurl) or [`graphql-inspector`](https://the-guild.dev/graphql/inspector):
+
+```bash title="introspecting the schema"
+export npm_config_yes=true
+
+# using graphqurl
+npx graphqurl https://graphql.union.build/v1/graphql \
+  --introspect > schema.graphql
+
+# some clients require a JSON schema
+npx graphqurl https://graphql.union.build/v1/graphql \
+  --introspect \
+  --format json > schema.json
+
+# using graphql-inspector
+npx @graphql-inspector/cli \
+  introspect https://graphql.union.build/v1/graphql \
+  --write schema.graphql
+```
+
+This schema can be used by API clients and editors like VSCode to provide a better developer experience.
+
+## Concepts
+
+The following definitions are used across the schema and documentation:
+
+- `source`: The chain or rollup where the packet is sent from, often prefixed such as `source_timestamp` or `source_transaction_hash`.
+- `destination`: The chain or roll-up that receives the packet. 
+- `sender`: the contract or [EOA](https://ethereum.org/en/developers/docs/accounts/) that made the transfer.
+- `receiver`: the contract or [EOA](https://ethereum.org/en/developers/docs/accounts/) that received the transfer.
+
+## Queries
+
+Below we have documented some common useful queries that are also leveraged in [app.union.build](https://app.union.build). For all queries, Union
+provides caches to increase performance. Use the syntax `@cached(ttl: ${time_in_seconds})` to cache the query. The cache key, which is used 
+to map the query to the response, is computed by hashing:
+
+- the GraphQL query.
+- the GraphQL operation name.
+- the GraphQL variables of the query.
+
+Caching can dramatically improve performance. Always add at least `@cached(ttl: 1)` to cache data between different windows, tabs, or users.
+
+### Transfers
+
+Transfers facilitated by Union (either `ucs-relay-1` or `ics20-1`) are queryable using v1_transfers.
```

Comment:
why -1 postfix? the standard is `UCS01-relay`

Reply:
No it's ucs-relay-1 in the version field of the channel

Reply:
The channel version of ucs01 relay is `ucs01-relay-1`, same as `ics20-1` is the version for ICS20 channels